### DISCUSSION TITLE
Github student approved still not getting benifits

### DISCUSSION THREAD

**Discussion by dj216185** [community/community#66476] [GitHub Education]
### Hi everyone,

"Congrats! Your request for @dj216185 was approved and your benefits will be available soon." this message is displayed but it is been 10 days since i applied . How to get the benefits ? 
![github approved](https://github.com/community/community/assets/121048276/6bddc26f-0b77-4ce7-8dc0-23d2b9c996d3)

**Comment by mantasu** [comment]
Hey, I had the same issue, the benefits should become active when "Approved" tag becomes purple:

![Screenshot 2023-09-24 072637](https://github.com/community/community/assets/56790921/d80b71bb-8d34-424d-a682-39a0f986389c)

If it's been over 2-3 weeks and you still haven't received them, just open a [ticket](https://support.github.com/request/education?tags=docs-generic).

**Reply by mantasu** [reply]
If you haven't received the benefits yet, I saw that staff is suggesting reapplying in some cases, for example [here](https://github.com/orgs/community/discussions/69133#discussioncomment-7185814), so worst case I would suggest to just re-raise the ticket and reapply :)

**Comment by advayc** [comment]
You'll get your benefits about 72 hours after your application. After then you'll get a email along with the PRO badge on your github profile.
